duckduckgo/chrome-zeroclickinfo
# reorder whitelist

**Reviewer:**



## Description:


Added two new whitelists. One tracker whitelist and one temporary broken site list. I also changed trackers.js to return the tracker data and a blocking decisions `block: true`. https://github.com/duckduckgo/chrome-zeroclickinfo/pull/189/files#diff-2650fb3e14b7f42d68a8766269c54434R169

## Steps to test this PR:

- Change the subdomain for the broken site list to `jason.duckduckgo.com` and reload the extensio. https://github.com/duckduckgo/chrome-zeroclickinfo/pull/189/files#diff-60e6512a6a0d9f504ae29abefe8cfc23R31
- reddit.com is listed as a broken site and tracker blocking should be turned off. I greyed out the popup in this case.

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
